ephios/settings.py

```diff
@@ -311,5 +312,25 @@ def GET_SITE_URL():
     "DEFAULT_AUTHENTICATION_CLASSES": [
         "rest_framework.authentication.TokenAuthentication",
         "rest_framework.authentication.SessionAuthentication",
+        "oauth2_provider.contrib.rest_framework.OAuth2Authentication",
     ],
 }
+
+# Like UserProfile, these models are implemented using djangos private swappable API
+# due to how swappable works, we need to list every model for it to work, even if we are not swapping all
```

Comment:
update the comment please. We not only need to list every model, but swap every model to our own version.